### ISSUE TITLE
Suggestions for deriving optimal cost matrix for use in Decision Trees

### ISSUE THREAD

**Issue by eashwarsiddharth** [sagitechls/SSN_SACE_2017_Jan#31]
Using a Cost Matrix will improve the efficiency, especially in case of class imbalance.

**Comment by Rajhan** [comment]
So you want to assign some penalty for every misclassified instance.

**Comment by eashwarsiddharth** [comment]
Yes, but, I'm not sure of a method to derive the optimal cost matrix. I have used arbitrary values with only slight improvement.
